src/Stryker.Core/Stryker.Core/TestRunners/VsTest/VsTestRunner.cs

```diff
@@ -100,12 +100,12 @@ public TestRunResult RunAll(ITimeoutValueCalculator timeoutMs, Mutant activeMuta
             return TestMultipleMutants(timeoutMs, activeMutant == null ? null : new List<Mutant> { activeMutant }, update);
         }
 
-        public TestRunResult TestMultipleMutants(ITimeoutValueCalculator timeoutCalc, IReadOnlyList<Mutant> mutants, TestUpdateHandler update)
+        public TestRunResult TestMultipleMutants(ITimeoutValueCalculator timeoutMs, IReadOnlyList<Mutant> mutants, TestUpdateHandler update)
```

Comment:
In this case the interface probably needs to be changed as the parameter type and name have been changed in the implementation but the interface param name was forgotten.

Reply:
Understood. The interface's method parameter was renamed to `timeoutCalc` and both implementations (`VsTestRunner` and `VsTestRunnerPool`) were updated accordingly.